py3status/modules/xrandr_rotate.py

```diff
@@ -57,24 +55,14 @@ class Py3status:
     def post_config_hook(self):
         self.displayed = ''
 
-    def _call(self, cmd):
-        process = Popen(cmd, stdout=PIPE, shell=True)
-        output = process.communicate()[0] or ""
-        try:
-            # python3
-            output = output.decode()
-        except:
-            pass
-        return output.strip()
-
     def _get_all_outputs(self):
         cmd = 'xrandr -q | grep " connected [^(]" | cut -d " " -f1'
-        return self._call(cmd).split()
+        return self.py3.command_output(cmd, shell=True).strip().split()
```

Comment:
I'm guessing not. I'm making it a simple conversion (carrying `strip()` from old `_call` method because of existing bugs that I still experience.. so I keep things simple.